src/Extension/Resources/config/services.yaml

```diff
@@ -1,3 +1,6 @@
+parameters:
+  uname_info: '%env(UNAME_INFO)%'
```

Comment:
Needs to have default param here it will be current system. like
UNAME_INFO ?: php_uname();